package.json

```diff
@@ -32,29 +32,32 @@
     "tidy:scripts": "eslint --fix test/** scripts/** '*.js' && prettier --write test/** scripts/** '*.js'",
     "size": "npx hardhat size-contracts",
     "coverage": "npx hardhat coverage",
-    "deploy-suite:hardhat": "npx hardhat clean && npx hardhat compile && npx hardhat run --network hardhat scripts/deploy-suite.js",
-    "deploy-suite:local": "npx hardhat clean && npx hardhat compile && npx hardhat run --network localhost scripts/deploy-suite.js",
-    "deploy-suite:test": "npx hardhat clean && npx hardhat compile && npx hardhat run --network test scripts/deploy-suite.js  >> logs/test.deploy.contracts.txt",
-    "deploy-suite:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mainnet scripts/deploy-suite.js >> logs/mainnet.deploy.contracts.txt",
-    "deploy-suite:polygon:mumbai": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mumbai scripts/deploy-suite.js >> logs/mumbai.deploy.contracts.txt",
-    "deploy-suite:polygon:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network polygon scripts/deploy-suite.js >> logs/polygon.deploy.contracts.txt",
+    "deploy-suite:hardhat": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network hardhat",
+    "deploy-suite:local": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network localhost",
+    "deploy-suite:test": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network test >> logs/test.deploy.contracts.txt",
+    "deploy-suite:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mainnet >> logs/mainnet.deploy.contracts.txt",
+    "deploy-suite:polygon:mumbai-test": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mumbai --env test >> logs/mumbai-test.deploy.contracts.txt",
+    "deploy-suite:polygon:mumbai-staging": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mumbai --env staging >> logs/mumbai-staging.deploy.contracts.txt",
+    "deploy-suite:polygon:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network polygon >> logs/polygon.deploy.contracts.txt",
     "verify-suite:ethereum:mainnet": "npx hardhat verify-suite --network mainnet --chain-id 1 --env mainnet  >> logs/mainnet.verify.contracts.txt",
     "verify-suite:polygon:mumbai-test": "npx hardhat verify-suite --network mumbai --chain-id 80001 --env mumbai-test >> logs/mumbai-test.verify.contracts.txt",
```

Comment:
I ran `npm run deploy-suite:polygon:mumbai-test`. The address file was created correctly. I then ran `npm run verify-suite:polygon:mumbai-test`. It finished very quickly without verifying anything. The only thing in the lig file was "🔍 Verifying contracts on block explorer..."
Usually even if the contract bytecode has already been verified, it takes a while and then it reports an error.

Reply:
I don't think this has anything to do with the changes I made.
I checked that `scripts/util/verify-suite.js` still reads file as expected.

Verify script accepts filter which tells which contracts should be verified and the default filter is an empty array, so it just skips all verifications.